src/app/layout.tsx

```diff
@@ -20,23 +23,22 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en" suppressHydrationWarning>
-      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-      <meta property="og:title" content="Babylon - Staking Dashboard" />
-      <meta name="description" content="BTC Staking Dashboard" key="desc" />
-      <meta property="og:description" content="BTC Staking Dashboard" />
-      <meta property="og:image:type" content="image/png" />
-      <meta property="og:image:width" content="2048" />
-      <meta property="og:image:height" content="1170" />
-      <meta property="og:image" content={`${getNetworkAppUrl()}/og.png`} />
-      <meta name="twitter:card" content="summary_large_image" />
-      <meta name="twitter:title" content="BTC Staking Dashboard" />
-      <meta name="twitter:description" content="BTC Staking Dashboard" />
-      <meta name="twitter:image" content={`${getNetworkAppUrl()}/og.png`} />
-      <meta name="twitter:image:type" content="image/png" />
-      <meta name="twitter:image:width" content="2048" />
-      <meta name="twitter:image:height" content="1170" />
+      <head>
+        <MetaTags />
```

Comment:
I think we can use next.js [default ](https://nextjs.org/docs/app/building-your-application/optimizing/metadata)approach for metadata instead of this

Reply:
Thanks for pointing that out. It would definitely work better for `layout.tsx` but I am afraid the `global-error.tsx` would have to do it this way though.

Reply:
I had to resolve this conversation to get the PR merge in but feel free to continue the discussion if you want to. :)